kani-compiler/src/codegen_cprover_gotoc/codegen/typ.rs

```diff
@@ -1240,8 +1239,17 @@ impl<'tcx> GotocCtx<'tcx> {
                                         variants,
                                         initial_offset,
                                     )
-                                }),
-                            ));
+                                });
+                            let cases_size = Size::from_bytes(cases.sizeof(&gcx.symbol_table));
```

Comment:
Could we get that information from rustc LayoutOf?

Reply:
Done. Not sure if the way I implemented it is the best though.